code/modules/mob/living/carbon/alien/humanoid/death.dm

```diff
@@ -11,4 +11,9 @@
 		update_icons()
 		status_flags |= CANPUSH
 
+		for (var/mob/living/carbon/alien/a in aliens)
+			a << "<span class='alertalien'>[src] has been slain at [src.loc.loc ? src.loc.loc : src.loc ? src.loc : "the void"]!</span>"
```

Comment:
src.loc.loc.loc.loc.loc.loc
AAAAAAAAAAAAAAAAAAAAAAAAAAA
what the hell is this btw, there's already code for the queen's death